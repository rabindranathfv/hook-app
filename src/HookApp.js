import React from 'react';

export const HookApp = () => {
  return (
    <div>
        <h1> HELLO HOOKS <i class="fas fa-h1"></i></h1>
    </div>
  )
}
