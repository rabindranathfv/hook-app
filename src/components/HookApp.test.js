import React from 'react';
import { shallow } from 'enzyme';
import { HookApp } from '../HookApp';

describe('HookApp Component::', () => {
    
    it('should render HookApp', () => {
        const wrapper = shallow(<HookApp />);
        expect(wrapper).toMatchSnapshot();
    });
    
})
