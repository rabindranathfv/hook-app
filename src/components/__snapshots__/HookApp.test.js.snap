// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HookApp Component:: should render HookApp 1`] = `
<div>
  <h1>
     HELLO HOOKS 
    <i
      class="fas fa-h1"
    />
  </h1>
</div>
`;
